src/features/Status/StatusPage.tsx

```diff
@@ -141,7 +153,7 @@ class StatusPage extends React.Component<IStatusPageProps, IStatusPageState> {
                     </Button>
                   </Flex>
                 </Flex>
-              ) : Date.now() > CONSTANTS.DECISION_CLOSE_TIME &&
+              ) : new Date() > new Date(this.state.settings.confirmTime) &&
```

Comment:
Interesting case here that even past close time, hacker status might change. This could warrant for settings for hackathon start and end times, or just end time to handle more edge cases that we could add later

Reply:
Can you elaborate more on these other edgecases?

Reply:
tbh kind of lost where I was going with this since it's been a while. One case I can think of is hacker status could be updated even after confirm time, like during the hackathon for walk ins